pkg/openstack/rabbitmq.go

```diff
@@ -52,7 +74,7 @@ func ReconcileRabbitMQ(ctx context.Context, instance *corev1beta1.OpenStackContr
 		// operator to continuously mutate the CR when setting it:
 		// https://github.com/rabbitmq/cluster-operator/blob/main/controllers/reconcile_operator_defaults.go#L19
 		if rabbitmq.Spec.Image == "" {
-			rabbitmq.Spec.Image = "rabbitmq:3.10.2-management"
+			rabbitmq.Spec.Image = "registry.redhat.io/rhosp-rhel9/openstack-rabbitmq:17.0"
```

Comment:
Eventually we'll want to make an ENV variable out of this and store the default in the CSV. But as I had this hard coded to begin with that can wait till later I think. No need to change it now.